pkg/airgap/update.go

```diff
@@ -301,3 +314,28 @@ func canInstall(beforeKotsKinds *kotsutil.KotsKinds, afterKotsKinds *kotsutil.Ko
 
 	return nil
 }
+
+func GetMissingRequiredVersions(appID string, airgap *kotsv1beta1.Airgap) ([]string, error) {
+	appVersions, err := store.GetStore().GetAppVersions(appID)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get downstream versions")
+	}
+
+	installedVersions := make(map[string]bool)
+	for _, appVersion := range appVersions {
+		if appVersion.ChannelID != airgap.Spec.ChannelID {
+			continue
+		}
+		installedVersions[appVersion.UpdateCursor] = true
+	}
+
+	missingVersions := make([]string, 0)
+	for _, requiredRelease := range airgap.Spec.RequiredReleases {
+		if installedVersions[requiredRelease.UpdateCursor] {
+			break
+		}
+		missingVersions = append(missingVersions, requiredRelease.VersionLabel)
```

Comment:
nit pick, Alex mentioned that these should be in ascending order, so i did this for online installs
```go
versionLabels = append([]string{v.VersionLabel}, versionLabels...)
```